api/chat_main_service.go

```diff
@@ -10,6 +10,7 @@ import (
 	"log"
 	"net/http"
 	"strings"
+	"time"
 
 	uuid "github.com/iris-contrib/go.uuid"
 	"github.com/samber/lo"
```

Comment:
:robot: OpenAI

This patch adds the `time` package to the imports. However, it is not used in this file. 

If you plan to use it later, then this patch is acceptable. Otherwise, it would be better to remove the unused import statement.

```diff
@@ -10,6 +10,7 @@ import (
 	"log"
 	"net/http"
 	"strings"
+	"time"
 
 	uuid "github.com/iris-contrib/go.uuid"
 	"github.com/samber/lo",
```

I suggest removing the unused import statement.

<!-- This is an auto-generated comment by OpenAI -->